wizeline/project-lab
# Feat/e2etest login

#### What does this PR do?

Create a way to log in as a test user.
Only a test@wizeline.com mail can be used.
Once logged in, store the state so it can be reused in other tests

#### Where should the reviewer start?

TestLogin page and playwright folder

#### How should this be manually tested?

Run the seed so the profile for test@wizeline is in DB
Download the .env (playwright) file from keybase and use the login information in it
http://localhost:3000/testLogin
Populate information and press login

Expected result:
the user can log in

Try with different credentials, the login should fail

#### Any background context you want to provide?

https://playwright.dev/docs/auth

#### What are the relevant tickets?

fixes #330
